webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityReactions.vue

```diff
@@ -81,6 +80,10 @@ export default {
       type: Number,
       default: 0
     },
+    kudosNumber: {
```

Comment:
In social layer, we don't have kudos, you will have to add an `<extension-registry-components` to allow its extensibility